.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,39 @@
+name: K8s external storage E2E Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  test:
+    name: E2E Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.13
+        id: go
+      - name: Build image 
+        run: |
+          make build
+          docker build -t nfsplugin:latest .
+      - uses: engineerd/setup-kind@v0.4.0
+        name: Installing KinD cluster
+        with:
+          version: "v0.9.0"
```

Comment:
What version of Kubernetes is this?

Reply:
This version of kind uses node image of Kubernetes v1.19.1